src/main/java/com/regnosys/testing/projection/ProjectionTestExtension.java

```diff
@@ -217,15 +200,14 @@ private ProjectionTestResult runProjection(Path projectExpectationFilePath,
             // No exceptions
             ExpectedAndActual<Boolean> error = new ExpectedAndActual<>(projectExpectationFilePath, expectation.isError(), false);
 
-            return new ProjectionTestResult(expectation.getInputFile(), expectation.getKeyValueFile(), expectation.getOutputFile(), keyValue, outputXml, validationFailures, validXml, error);
+            return new ProjectionTestResult(expectation.getInputFile(), expectation.getKeyValueFile(), expectation.getOutputFile(), null, outputXml, validationFailures, validXml, error);
```

Comment:
Let's remove `ProjectionTestResult.keyValue` so you don't have to pass in null

Reply:
would not need it as we are not reoving the files from model as yet Minesh said not to delete them yet